src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -33,14 +31,10 @@ import { useUser } from "hooks/useUser";
 import { useVenueEvents } from "hooks/events";
 
 import { Button } from "components/atoms/Button";
-import { Schedule } from "components/molecules/Schedule";
+import { ScheduleNG } from "components/molecules/ScheduleNG";
 import { ScheduleVenueDescription } from "components/molecules/ScheduleVenueDescription";
 
-import {
-  buildLocationString,
-  extractLocation,
-  prepareForSchedule,
-} from "./utils";
+import { prepareForSchedule } from "./utils";
```

Comment:
the function `prepareForSchedule` should be modified as well.
for the correct display of the event on the timeline, I modified start/end time of the event to be it within the selected date for the events which are started/ended out of the date boundaries.
Now we don't need that. Moreover, it'd be better to show the actual start/end date if it does not matches the selected date.

**Example:** the event starts at 11pm today and ends at 2am tomorrow.
then if we select "today" we show:
> 11pm - tomorrow 2am 

and if we have "tomorrow" selected, we show
> today 11pm - 2am

Reply:
fixed in 4aab2820998c17b1dcee767fd370f0f688357cb7

Modified. The events are displayed in a next way. WDYT? 
<img width="486" alt="Screenshot 2021-08-04 at 11 14 47" src="https://user-images.githubusercontent.com/26366773/128150014-4e6b152f-2842-4cb3-8236-932d437d5376.png">
<img width="489" alt="Screenshot 2021-08-04 at 11 15 11" src="https://user-images.githubusercontent.com/26366773/128150024-dc7cd0c1-0171-48e6-9767-77c8c3977787.png">

Reply:
discussed internally and fixed in 36debcd7850a46887963787bcc257810b4e6e2f1